config/environments/production.rb

```diff
@@ -46,8 +46,8 @@
   config.force_ssl = true
 
   # Log to STDOUT by default
-  config.logger = ActiveSupport::Logger.new($stdout)
-    .tap { |logger| logger.formatter = config.log_formatter }
+  # config.logger = ActiveSupport::Logger.new($stdout)
+  #   .tap { |logger| logger.formatter = config.log_formatter }
```

Comment:
I'd prefer if we kept the logging configuration in `config/environments/production.rb`, so that it's closer to what is in the default Rails configuration.

Or, if we don't want to do that, we should probably delete these lines rather than commenting them out.

Reply:
The config should be the same for production and development, which is why I've put the configuration in `application.rb` rather than here.

I can delete these lines rather than comment them out - I saw that we deliberately comment out a lot of the defaults to make it clear that we don't want to use those defaults so wasn't sure what to do here. If you think it's better to delete rather than comment our I can do that.

Reply:
My view is we should have local less noisy and if we want it to be like production, say for debugging, an individual can easily flip that on. Plus keeping things closer to Rails means when we do our next upgrade we've deviated less.

Reply:
Ok sure - happy to go with whatever the general preference is, so would your suggestion be to have the default logger configured for development, and only have the logger that adds the context for production?

How about the Lograge request logging, should I keep that how it is in `application.rb` or move that too?

Reply:
Yes I think so on the first one. I think on the second one feels that we should move out for consistency.